### PR TITLE
Validate docs and test links from travis

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

**Comment by muffinresearch** [comment]
r+

File: docs/topics/api.rst [hunk 1]
```diff
@@ -182,7 +182,7 @@ does not require user authentication.
 
 
 .. _`retrieve a token`: https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate
-.. _`token generator`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#generate-a-token
+.. _`token generator`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#post--braintree-token-generate-
```
Comment (tofumatt) [inline docs/topics/api.rst:185]: The dashes here look weird, are they correct? 😬

Comment (kumar303) [inline docs/topics/api.rst:185]: yeah, they are definitely weird. I guess sphinx auto-generates anchors like this.

Comment (tofumatt) [inline docs/topics/api.rst:185]: Ah, yes, Sphinx makes weird URLs.
- tofumatt

**Pull request merged**